src/Utility/ParatextHelpers.cs

```diff
@@ -0,0 +1,216 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using SIL.Transcriber.Models;
+using SIL.Transcriber.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace SIL.Transcriber.Utility
+{
+    public class ParatextHelpers
+    {
+        public static string ParatextProject(int projectId, ProjectService projectService)
+        {
+            var paratextSettings = projectService.IntegrationSettings(projectId, "paratext");
+            if ((paratextSettings ?? "") == "")
+            {
+                throw new Exception("No Paratext Integration Settings for this project " + projectId.ToString());
+            }
+            dynamic settings = JsonConvert.DeserializeObject(paratextSettings);
+            return settings.ParatextId;
+        }
+
+        public static XElement GenerateParatextData(Passage currentPassage, XElement chapterContent, string transcription, string heading = "")
+        {
+            var verses = chapterContent.Nodes().Where(n => n.NodeType == System.Xml.XmlNodeType.Element && ((XElement)n).Name.LocalName == "verse");
+
+            if (chapterContent.Descendants("chapter").Count() == 0)
+            {
+                //chapterContent.Add()
+            }
+            var verse = verses.Where(n => ((XElement)n).FirstAttribute.Value == currentPassage.StartVerse + "-" + currentPassage.EndVerse).FirstOrDefault();
+            if (verse != null)
+            {
+                if (verse.NextNode.NodeType != System.Xml.XmlNodeType.Text)
+                    verse.AddAfterSelf(new XText(""));
+                ((XText)verse.NextNode).Value = transcription;
+            }
+
+            /*
+             var startContains = verses.FirstOrDefault(n => n.  f => f.TrimStart().StartsWith(currentPassage.StartVerse.ToString() + " ")
```

Comment:
Commented code should be taken out.

Reply:
It's a work in progress -- currently coded to work exactly for the demo.  Some of the commented code will come back.